### ISSUE TITLE
Feature request : enable drag-n-drop notes to ftags/folder

### ISSUE THREAD

**Issue by antoineschmitt** [standardnotes/forum#179]
When I want to Tag a note, I need to select it and manually add the tag to it, by typing the tag.
This is cumbersome.
It would be really nice to be able to drag-n-drop a note to a tag to assign it to it.
It would even be better to be able to select multiple notes and drag them all to one tag, in order to assign the tag to the notes.

Same for "folders" (when the Folders extension is installed) : be able to drag one or multiple notes to a folder to assign the folder to the note(s).

**Comment by mobitar** [comment]
The philosophy we ascribe to is, it's better to be just a little bit cumbersome, than make everything easy with additional features, causing bloat and bugs. In this case, this would just be another feature we would have to maintain.

**Comment by cormacrelf** [comment]
It seems like you guys are using AngularJS? If you ever make the big move to Angular latest, adding rock-solid drag and drop features like this would be quite sane and manageable. I wrote a library for it.

https://cormacrelf.github.io/angular-skyhook/index.html

**Comment by jmichaels** [comment]
@mobitar - I totally understand not wanting to add code bloat and keep the code base lean.  That being said, being able to perform certain operations on multiple notes at the same time seems like a really important feature to me.

All of these apps have this feature:  Bear (an even more minimalist competitor), Apple Notes, Evernote, Google Keep, Microsoft OneNote.

I just tried to import my 1000+ library of notes, and I had an existing tagging scheme.  Because I can't tag multiple notes at the same time I'm going to have to re-tag 1,000 notes one at a time by hand.  It's going to take me days of work to get everything retagged when it would take me an hour in another app.

Also, even if I got everything imported, I could never reorganize because it would be way too much of a hassle.

This is probably going to be a deal breaker for me, as much as I love this app and your team's commitment to privacy and openness.

**Comment by jmichaels** [comment]
@mobitar - If "Drag and Drop" stuff is what you're trying to avoid (definitely don't blame you), this would also solve my problem:

* Select multiple notes
* Click an "Add tag" button or menu option
* Modal window appears, select the tag from a drop down list of your existing tags

**Comment by mobitar** [comment]
I don't mind building custom extensions for stuff like this, but definitely wouldn't build something like this into the core application. Standard Notes as you see it right now is complete. I will try my hardest not to add anything new, even if it turns away great users.

But, as for an extension that allows you to tag multiple notes, well that's totally on the table. But I just don't have the capacity for it right now.

**Comment by jmichaels** [comment]
That's entirely fair, thanks for the response and explanation.

**Comment by jedypod** [comment]
I just signed up for a year subscription to Standard Notes and I've got to say I agree with the sentiment in this thread. I'm very excited about the feature set and philosophy of this application, keeping the code base lean is not a valid excuse for forcing a user to be unable to sort and organize notes in batches. This is a big giant hole in functionality for me right now.

When importing my notes from another application, I seriously have to manually select each note individually and add a specific tag or folder to that note, one by one?

Is there no workaround for this? What is the suggested workflow?

Thanks for your time!
